driver/norma/main.go

```diff
@@ -40,6 +41,13 @@ func main() {
 			&diffCommand,
 		},
 	}
+
+	app.Flags = append(app.Flags, globalflags.AllLoggerFlags...)
+
+	app.Before = func(c *cli.Context) error {
```

Comment:
nit: both of these could be set up directly in the struct initialization above, right?